src/data/resolvers/colonyActions.ts

```diff
@@ -135,12 +137,15 @@ export const colonyActionsResolvers = ({
           colonyClient as ColonyClient,
           actionType,
         );
-
-        const annotation = await getAnnotation(
-          from as string,
-          hash as string,
-          colonyClient as ColonyClientV5,
-        );
+        const clientVersion = await colonyClient?.version();
+        let annotation;
+        if (clientVersion.toString() === '5') {
```

Comment:
Please don't hard-code the version number.

Instead use the version provided by colonyJS via `ColonyVersion`

```suggestion
        if (clientVersion === ColonyVersion.CeruleanLightweightSpaceship) {
```